scriptmodules/libretrocores/lr-pcsx-rearmed.sh

```diff
@@ -29,13 +29,15 @@ function sources_lr-pcsx-rearmed() {
 function build_lr-pcsx-rearmed() {
     local params=(THREAD_RENDERING=0)
 
-    if isPlatform "arm"; then
+    if isPlatform "arm" ; then
         params+=(ARCH=arm DYNAREC=ari64)
-        if isPlatform "neon"; then
-            params+=(HAVE_NEON=1 BUILTIN_GPU=neon)
-        else
-            params+=(HAVE_NEON=0 BUILTIN_GPU=peops)
-        fi
+    elif isPlatform "aarch64" ; then
+        params+=(ARCH=aarch64 DYNAREC=ari64)
+    fi
+    if isPlatform "neon"; then
+        params+=(HAVE_NEON=1 BUILTIN_GPU=neon)
+    else
+        params+=(HAVE_NEON=0 BUILTIN_GPU=peops)
```

Comment:
Why was this block moved outside of the arm logic ?

Reply:
I didn't think it would ever get triggered outside of the arm logic, so I figured it was better to leave it outside than to put duplicate lines in for both 32-bit and 64-bit

Reply:
Yes, sorry - that makes sense or else you would need to add `params+=(HAVE_NEON=0 BUILTIN_GPU=peops)` again.

Apart from the unrelated whitespace change I'm happy so if you can change that and squash the commits I will merge. Thanks.